src/test/java/io/confluent/idesidecar/restapi/integration/connection/DirectConnectionSuite.java

```diff
@@ -14,12 +15,98 @@
 import io.confluent.idesidecar.restapi.integration.ITSuite;
 import io.confluent.idesidecar.restapi.models.Connection;
 import io.confluent.idesidecar.restapi.models.ConnectionSpec.ConnectionType;
+import io.confluent.idesidecar.restapi.models.ConnectionStatus.Authentication.Status;
 import io.confluent.idesidecar.restapi.models.ConnectionStatus.ConnectedState;
+import io.restassured.http.ContentType;
 import jakarta.ws.rs.core.MediaType;
 import org.junit.jupiter.api.Test;
 
 public interface DirectConnectionSuite extends ITSuite {
 
+  @Test
+  default void shouldTestDirectConnection() {
+    // Not all environments support direct connections
+    var spec = environment().directConnectionSpec().orElse(null);
+    assertNotNull(spec, "Expected environment %s has direct connection spec".formatted(environment().name()));
+
+    // Test the connection and mark it as the one we'll use
+    var rsps = testConnectionWithResponse(spec)
+        .statusCode(200)
+        .contentType(ContentType.JSON)
+        .body("api_version", equalTo("gateway/v1"))
+        .body("kind", equalTo("Connection"))
+        .body("metadata.self", startsWith("http://localhost:26637/gateway/v1/connections/"))
+        .body("metadata.resource_name", nullValue())
+        .body("id", equalTo(spec.id()))
+        .body("spec.id", equalTo(spec.id()))
+        .body("spec.name", equalTo(spec.name()))
+        .body("spec.type", equalTo(ConnectionType.DIRECT.name()))
+        .body("status.authentication.status", equalTo(Status.NO_TOKEN.name()))
+        .body("status.kafka_cluster.state", equalTo(ConnectedState.SUCCESS.name()));
```

Comment:
I've come to really like this style of verification of the response objects. IMO, it's very clear as to what the test is asserting.